code/teams/TraitorTeam.cs

```diff
@@ -0,0 +1,13 @@
+namespace TTTReborn.Teams
+{
+    [TeamAttribute("Traitors")]
+    public class TraitorTeam : TTTTeam
+    {
+        public override Color Color => Color.FromBytes(223, 41, 53);
```

Comment:
Can we just use regular R, G, B values here?

Reply:
it is RGB. What do you mean? :)